Userland/Applications/Browser/CookieJar.h

```diff
@@ -1,40 +1,47 @@
 /*
- * Copyright (c) 2021, Tim Flynn <trflynn89@serenityos.org>
+ * Copyright (c) 2021-2022, Tim Flynn <trflynn89@serenityos.org>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
 
 #include <AK/DeprecatedString.h>
+#include <AK/Function.h>
 #include <AK/HashMap.h>
 #include <AK/Optional.h>
-#include <AK/Traits.h>
 #include <LibCore/DateTime.h>
 #include <LibWeb/Cookie/Cookie.h>
 #include <LibWeb/Forward.h>
 
 namespace Browser {
 
-struct CookieStorageKey {
-    bool operator==(CookieStorageKey const&) const = default;
-
-    DeprecatedString name;
-    DeprecatedString domain;
-    DeprecatedString path;
-};
+class Database;
 
 class CookieJar {
+    struct Statements {
+        u64 create_table { 0 };
+        u64 insert_cookie { 0 };
+        u64 update_cookie { 0 };
+        u64 expire_cookie { 0 };
+        u64 select_cookie { 0 };
+        u64 select_all_cookies { 0 };
+    };
```

Comment:
Nitpick (and/or "would be nice"): These could use a more...distinct...type, I initially thought they were statistics.

Reply:
Gave all the `u64`s proper names